File: solutions/java/q1-50/AvgNumberInList.java
/*
Created on Nov 20, 2019

@author: Ashish Namdev
*/

/*
## Q25
### Can you write a program to find the average of numbers in a list / array?
*/

import java.util.Arrays;

public class AvgNumberInList
{
    public static void main(final String []args)
    {
        int n = 10;
        int [] a = new int[n];
        int sum = 0;
        double avg = 0;

        for (int i = 0; i < n; i++)
        {
            a[i] = i + 1; 
        }
        
        for(int i = 0; i < a.length; i++)
        {
            sum += a[i];
        }

        avg = (sum / a.length);

        System.out.printf("Average of elements in the %s = %.2g\n", Arrays.toString(a), avg);
    }
}

File: solutions/python/avg_number_in_list.py
'''
Created on Nov 17, 2019

@author: Ashish Namdev
'''

## Q7 Programming
### Can you write a program to find the average of numbers in a list / array?

VERSION = '1.0.0'
DATE = '2019-11-17'
UPDATED = '2019-11-17'

n = 10
a = [i for i in range(1, n + 1)]
avg = sum(a)/n
print("Average of elements in the %s = %.1f" % (a, avg))

